src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -334,4 +329,26 @@
     }
 }
 // Theming
-build-themes-prefsEditor(themes);
+[class*="fl-theme"]:not(.fl-theme-prefsEditor-default) {
+    .fl-prefsEditor {
+        // ON/OFF switch
+        .fl-switchUI-control:focus {
+            box-shadow: 0 0 0 0.2em var(--fl-bgColor),
+                0 0 0 0.35em var(--fl-fgColor);
+        }
+
+        .fl-switchUI-controlKnob {
+            background-color: var(--fl-fgColor) !important;
+        }
```

Comment:
@greatislander the switch component is actually an Infusion component that can be used outside of the prefs editor. ([Switch Demo](https://build.fluidproject.org/infusion/demos/switch/)). With this change of moving the theming into the PrefsEditor styles, we'd need to have a prefs editor on the same page for the switches to be themed. However, there is a possibility that the page uses the UI Enhancer to style the page but does not include the prefs editor on the page. An example of that can be seen in the full page prefs editors ([example](https://build.fluidproject.org/infusion/examples/framework/preferences/minimalFootprint/)), where the preference editor is located on a separate page. This might be for performance reasons, moving the preferences to a user account page, and etc. I think for now we should move these styles back into the Themes.styl file, that way they are always available with the enactors. However, this may still not be the best factoring and we should consider how to improve things, with the work on [FLUID-6552](https://issues.fluidproject.org/browse/FLUID-6552).

Reply:
@jobara If I add fallbacks (`#fff` for `--fl-bgColor` and `#000` for `--fl-fgColor`), the switch component will work in its default styling without any theme being applied. Let me know if that would work to address this issue.

Reply:
@greatislander I don't think so. I believe the issue is a bit different. It's the case of theming the switch components in the absence of the prefs editor. The enacting and setting of preferences can happen independently.

Reply:
Resolved this in https://github.com/fluid-project/infusion/pull/1007/commits/94eacc6cb898acf9b3af6f619dd38bad63e683c6.

Reply:
Added this: https://issues.fluidproject.org/browse/FLUID-6553